css/main.css

```diff
@@ -0,0 +1,1109 @@
+html {
+    padding: 0;
+    margin: 0;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+body {
+    display: flex;
+    min-height: 100px;
+    background-color: #000;
+    margin: 0;
+    padding: 0;
+}
+
+.logo {
+    height: 20px;
+    width: 20px;
+}
+
+.logo img {
+    position: absolute;
+    height: 70px;
+    width: 70px;
+    left: 40px;
+    top: 18px;
+}
+
+.nav {
+    display: flex;
+    align-items: flex-end;
+    position: absolute;
+    top: 48px;
+}
+
+.nav_right-20px {
+    right: 300px;
+}
+
+.nav__list {
+    margin: 0;
+    padding: 0;
+    display: block;
+    list-style-type: none;
+    top: 48px;
+    font-size: 16px;
+    line-height: 24px;
+}
+.nav__list li > a {
+    color: #FFF;
+    text-decoration: none;
+}
+
+.nav__list li {
+    float: left;
+    margin: 0 50px 0 0;
+}
+
+.nav__list li:last-child {
+    margin-right: 0;
+}
+
+
+ .login > button {
+    display: block;
+     position: absolute;
+    background: linear-gradient(94.26deg, #C89AFC 9.51%, #7C6AFA 90.23%);
+    border-radius: 50px;
+    width: 183px;
+    height: 44px;
+    font-size: 16px;
+    text-align: center;
+    float: right;
+    right: 30px;
+    top: 48px;
+    justify-content: center;
+    align-items: center;
+}
+
+.tagline {
+    position: absolute;
+    width: 920px;
+    height: 180px;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 64px;
+    left: 160px;
+    top: 170px;
+    line-height: 90px;
+    white-space: nowrap;
+    text-align: left;
+    color: #FFF;
+}
+
+.timer {
+    display: flex;
+    flex-direction: row;
+    color: whitesmoke;
+    align-items: center;
+    margin: auto;
+}
+
+.timer__days {
+    background: #121212;
+    width: 107px;
+    height: 112px;
+    padding: 20px 30px;
+    top: 500px;
+    left: 250px;
+    display: flex;
+    position: absolute;
+    align-items: center;
+    margin-right: 50px;
+}
+
+.timer__hours {
+    background: #121212;
+    width: 107px;
+    height: 112px;
+    padding: 20px 30px;
+    top:500px;
+    left: 450px;
+    display: flex;
+    position: absolute;
+    align-items: center;
+    margin-right: 50px;
+}
+
+.timer__minutes {
+    background: #121212;
+    width: 107px;
+    height: 112px;
+    padding: 20px 30px;
+    top: 500px;
+    left: 650px;
+    display: flex;
+    position: absolute;
+    align-items: center;
+    margin-right: 50px;
+}
+
+.timer__seconds {
+    background: #121212;
+    width: 107px;
+    height: 112px;
+    padding: 20px 30px;
+    top: 500px;
+    left: 850px;
+    display: flex;
+    position: absolute;
+    align-items: center;
+    margin-right: 50px;
+}
+
+.main_screen {
```

Comment:
Please, fix delimiters everywhere. Now it seems like element `screen` for block `main`.